_posts/2023-09-07-r-project-sprint/r-project-sprint.Rmd

```diff
@@ -0,0 +1,194 @@
+
+* Joining the [R Contributors slack group](https://contributor.r-project.org/slack){target="_blank"}.
+
+* Participating in [online events](https://contributor.r-project.org/events/){target="_blank"} such as a C book club for R Contributors and R Contributor office hours.
+
+* Watching recorded [tutorials and demonstrations](https://contributor.r-project.org/tutorials/){target="_blank"}.
+
+* Submitting [proposals](https://contributor.r-project.org/r-project-sprint-2023/projects/ideas-and-proposals.html){target="_blank"} for topics to tackle during the Sprint.
+
+
+# Participants
+
+The [R Project Sprint participants](https://contributor.r-project.org/r-project-sprint-2023/participants/){target="_blank"} were impressively diverse both in terms
+of professional experience and geographic residence. A non-exhaustive list of countries
+represented at the in-person event includes Argentina, Brazil, Canada, England, India, Hungary, Nepal, Netherlands, New Zealand, Nigeria, Oman, and Senegal, Switzerland, and United States.
+
+Some individuals participated remotely via video conference, either by choice
+or circumstance. The combination of [the United Kingdom's air traffic
+control issues](https://www.bbc.com/news/uk-66637156){target="_blank"} and [train
+strikes](https://www.bbc.com/news/business-66479059){target="_blank"} caused delayed
+arrival for some.
+
+
+# During the event
+
+The issues addressed at the event fell into four broad categories:
+
+1. [Translation](https://contributor.r-project.org/r-project-sprint-2023/projects/translation/){target="_blank"} of messages, errors, and warnings.
+
+2. Improving [documentation](https://contributor.r-project.org/r-project-sprint-2023/projects/documentation/){target="_blank"}.
+
+3. Addressing bug reports, such as those related to the [`stats` package](https://github.com/r-devel/r-project-sprint-2023/issues/33){target="_blank"}.
+
+4. Enhancements, such as [logging visual outcomes from base graphics calls](https://github.com/r-devel/r-project-sprint-2023/issues/75){target="_blank"}. 
+
+# Knowledge requirements for contributions
+
+Depending on the task at hand, the technical requirements for contributions
+included:
+
+* Read R source code and understand function implementation (documentation and
+translation).
+
+* Read, modify, or write R source code (bugs and enhancements).
+
+* Read, modify, or write C source code (bugs and enhancements).
+
+# Workflows for contributions
+
+
+During the sprint, the progress on these items was tracked [https://github.com/r-devel/r-project-sprint-2023/issues](https://github.com/r-devel/r-project-sprint-2023/issues){target="_blank"}. Either the initial report or the final resolution of these items may be seen in the R Bug Tracking System, known as [Bugzilla](https://bugs.r-project.org/){target="_blank"}.
```

Comment:
was tracked _at_ <URL>? or did you mean to turn a word into a link here?

Reply:
yep, i meant at